src/components/DocumentationTopic.vue

```diff
@@ -622,6 +622,7 @@ export default {
     }
 
     this.store.reset();
+    this.store.setReferences(this.references);
```

Comment:
Sent you more details offline, but we may need to still consider having some way of also doing this as part of the update cycle and not just the initial created hook to support situations where the references may change during the lifetime of the component.

This does seem to fix the rendering issue that we noticed with references in bulleted lists though! Seems like you may have found that there is a race condition with the original way we're setting these refs causing that.